docs/src/univariate.md

```diff
@@ -418,6 +418,13 @@ Semicircle
 plotdensity((-1, 1), Semicircle, (1,)) # hide
 ```
 
+```@docs
+SkewedExponentialPower
+```
+```@example plotdensity
+plotdensity((-8, 5), SkewedExponentialPower, (0, 1, 0.7, 0.7)) # hide
+```
```

Comment:
For anyone who watches this PR: I built the documentation locally and the plot seems fine.
![image](https://user-images.githubusercontent.com/26102/143660163-a55bc371-5232-45dd-855f-3b66f5a5ce33.png)